functions/openhabian.bash

```diff
@@ -9,6 +9,16 @@ get_git_revision() {
   echo "[$branch]$latesttag-$revcount($shorthash)"
 }
 
+apt_update() {
+  echo -n "$(timestamp) [openHABian] Updating Linux package information ... "
+  apt-get -q update >/dev/null 2>&1 &
+  PID_APT=$!
+}
+
+wait_for_apt_to_finish_update() {
+    wait ${PID_APT} 2>/dev/null
```

Comment:
Pls make sure that PID_APT is set, otherwise call apt_update. Then we can still source the function files directly from shell without changing the experience (wait w/o a pid will just wait for child tasks, if there are none, it will continue).

Reply:
did you think of using `wait -f``?

Reply:
Might be a bit strange for users: a) it might wait for long without showing a reason b) on connection errors we are hiding the reason....

Reply:
ok now check for PID_APT and used wait -f
On your last point I would leave it like that. IIRC it does not work without that redirection so that's the lesser pain point (and only when Inet is down).